.gitignore

```diff
@@ -169,3 +169,9 @@ copier_scripts/
 
 # DS_Store files
 .DS_Store
+
+# Node Modules
+node_modules/
+
+# VSCode
+.vscode/
```

Comment:
We should add DS folders (like `templates/components`, `templates/layout`) and static assets directory here? Since we don't want to add DS files.

Reply:
Ok, yeah, wasn't sure about it so I preferred to ask.